sleap_nn/architectures/model.py

```diff
@@ -179,11 +179,11 @@ def from_config(
 
     def forward(self, x: torch.Tensor) -> torch.Tensor:
         """Forward pass through the model."""
-        if self.input_expand_channels != 1:
-            input_list = []
-            for _ in range(self.input_expand_channels):
-                input_list.append(x)
-            x = torch.concatenate(input_list, dim=-3)
+        # if self.input_expand_channels != 1:
+        #     input_list = []
+        #     for _ in range(self.input_expand_channels):
+        #         input_list.append(x)
+        #     x = torch.concatenate(input_list, dim=-3)
```

Comment:
Is this getting handled elsewhere now? If so, delete commented chunk

Reply:
This is not handled anywhere yet (added a TODO [here](https://github.com/talmolab/sleap-nn/blob/2f0fedc84ade1f906a81890a6ccf0e5e3195cec0/sleap_nn/training/model_trainer.py#L578)). I'll create a separate PR for handling different input channels